dom.js

```diff
@@ -0,0 +1,33 @@
+var APP = APP || {};
+(function(app) {
+
+    function element(name) {
+        return function() {
+            var e = document.createElement(name);
+            var args = Array.prototype.slice.call(arguments);
```

Comment:
Don't use the argument array like that, it's not readable. `function(element, attrObj, textContent)` is much more clear.